paper-api/src/main/java/org/bukkit/block/TrialSpawner.java

```diff
@@ -4,14 +4,41 @@
 import org.bukkit.entity.Entity;
 import org.bukkit.entity.Player;
 import org.bukkit.spawner.TrialSpawnerConfiguration;
-import org.jetbrains.annotations.ApiStatus;
 import org.jetbrains.annotations.NotNull;
 
 /**
  * Represents a captured state of a trial spawner.
  */
 public interface TrialSpawner extends TileState {
 
+    /**
+     * Gets the Gametime in ticks when the cooldown ends. 0 if not currently in cooldown.
+     *
+     * @return the Gametime in ticks
+     */
+    public long getCooldownEndsAt();
```

Comment:
Not a big fan of these method names, they feel *really* spigot.
`getCooldownEnd` is probably better.